### DISCUSSION TITLE
v1.2.2 Install speed, build preview, nsfw filter option

### DISCUSSION THREAD

**Discussion by glucauze** [glucauze/sd-webui-faceswaplab#48] [Announcements]
Try to improve install and start speed.

If --use-cpu is not use will install gpu requirements by default. Note that GPU is not activated by default.

add nsfw threshold option (1==disable) for performance reason. Use responsibly. There are now too many nsfw disabling options for the measures taken here to be effective.

Fix build preview

<hr /><em>This discussion was created from the release <a href='https://github.com/glucauze/sd-webui-faceswaplab/releases/tag/1.2.2'>Merge pull request #46 from glucauze/v1.2.2</a>.</em>

**Comment by LucianoDaluz** [comment]
Any up date on the horizon?
